scripts/bump_csi_version.sh

```
@@ -0,0 +1,172 @@
+        *) echo "Unknown parameter passed: $1"; print_help; exit 1 ;;
+    esac
+    shift
+done
+
+if [ "$1" == "--help" ] || [ "$1" == "-h" ]; then
+    print_help
+    exit 0
+fi
+
+if ! is_correct_version "$csi_plugin_version"; then
+    echo "Invalid csi_plugin_version format: $csi_plugin_version1"
```

Shellcheck (0.10.0)

[warning] csi_plugin_version1 is referenced but not assigned (did you mean 'csi_plugin_version'?).

(SC2154)